packages/charting/configure/src/correct-response.jsx

```diff
@@ -235,7 +237,7 @@ export class CorrectResponse extends React.Component {
                 domain={domain}
                 range={range}
                 charts={charts}
-                data={categories}
+                data={correctResponseCategories}
```

Comment:
This only renders a correct response, but I don't think this sets the correct response on the model. So let's make sure we have that.